gradle/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -174,6 +174,10 @@ class SentryPlugin implements Plugin<Project> {
                         mappingFile
                     ]
 
+                    if (!project.sentry.autoUpload) {
+                        args.push("--no-upload")
```

Comment:
Dumb question (mostly for my own education), what's the point of running the command at all if they don't want to upload? Is it just for users that might want to do a manual upload later or...?